comp/core/status/render_helpers.go

```diff
@@ -126,22 +127,34 @@ func lastErrorMessage(value string) string {
 }
 
 // formatUnixTime formats the unix time to make it more readable
-func formatUnixTime(unixTime int64) string {
+func formatUnixTime(unixTime any) string {
 	// Initially treat given unixTime is in nanoseconds
-	t := time.Unix(0, int64(unixTime))
-	// If year returned 1970, assume unixTime actually in seconds
-	if t.Year() == time.Unix(0, 0).Year() {
-		t = time.Unix(int64(unixTime), 0)
+	parseFunction := func(value int64) string {
+		t := time.Unix(0, value)
+		// If year returned 1970, assume unixTime actually in seconds
+		if t.Year() == time.Unix(0, 0).Year() {
+			t = time.Unix(value, 0)
+		}
+
+		_, tzoffset := t.Zone()
+		result := t.Format(timeFormat)
+		if tzoffset != 0 {
+			result += " / " + t.UTC().Format(timeFormat)
+		}
+		msec := t.UnixNano() / int64(time.Millisecond)
+		result += " (" + strconv.Itoa(int(msec)) + ")"
+
+		return result
 	}
 
-	_, tzoffset := t.Zone()
-	result := t.Format(timeFormat)
-	if tzoffset != 0 {
-		result += " / " + t.UTC().Format(timeFormat)
+	switch v := unixTime.(type) {
+	case int64:
+		return parseFunction(int64(v))
```

Comment:
Can remove the `int64` cast here. I think it would be better if only the `float64` case had the cast as it stands out more.